dotnet/docfx.json

```diff
@@ -55,12 +55,15 @@
       "breadcrumb_path": "/dotnet/roslyn_dotnet_toc/breadcrumb/toc.json",
       "apiPlatform": "dotnet",
       "author": "dotnet-bot",
```

Comment:
@sfilipi here's where I was asking if we should put you or another one of the devs to get tagged on new issues for ML.NET API docs?

Reply:
You can add me for it. How does one do that, update the author?

Reply:
I'll do that. I'll just add your GitHub alias to this file. 😄